### PR TITLE
Monorepo : added database-mongo to the monorepo

### PR DESCRIPTION
**Pull request by Aetherall**
Used `lerna import <mongo repository>`

Also added circleci/mongo to the circleCI docker images

### CODE REVIEW

File: packages/database-mongo/package.json [hunk 1]
```diff
@@ -32,38 +22,24 @@
...
   },
-  "homepage": "https://github.com/js-accounts/mongo",
+  "homepage": "https://github.com/accounts-js/accounts",
   "devDependencies": {
     "@accounts/common": "0.1.0-beta.3",
-    "@accounts/server": "0.1.0-beta.3",
-    "@accounts/tslint-config-accounts": "0.0.6",
-    "@types/jest": "22.2.0",
-    "@types/lodash": "4.14.104",
-    "@types/mongodb": "3.0.7",
```
Comment (pradel) [inline packages/database-mongo/package.json]: you removed some needed types

Comment (Aetherall) [reply packages/database-mongo/package.json]: I red that if a package exports it's types, it is not needed to include his typing package in devDeps

Comment (pradel) [reply packages/database-mongo/package.json]: is mongo exporting his types directly?

Comment (pradel) [reply packages/database-mongo/package.json]: lodash is not exporting it's types

Comment (Aetherall) [reply packages/database-mongo/package.json]: Seems to not be... I don't understand why tsc didn't crash then ...

Comment (pradel) [reply packages/database-mongo/package.json]: it's because we are not using strict mode, will be great to have but will need some work before

Comment (Aetherall) [reply packages/database-mongo/package.json]: I see, actually I am struggling with this, those typings mess up the compile function...

Comment (Aetherall) [reply packages/database-mongo/package.json]: Is it needed to also revert @accounts/tslint-config-accounts and @types/jest ?

Comment (pradel) [reply packages/database-mongo/package.json]: In that case no :)

**Comment by davidyaha** [comment]
@pradel  @Aetherall is it good to go?

**Comment by pradel** [comment]
My latest commit on the mongo repo is missing (unset function)

**Comment by Aetherall** [comment]
Hum... my git talents are pretty limited... Is there a nice way to get your most recent commits ?

**Comment by davidyaha** [comment]
I thing create a new branch from master, re-do the `lerna import` and then cherrypick your added commits (the ones that fixes the build system and whatnot)

**Comment by Aetherall** [comment]
Ah okay, I thought it would be a way that kept the PR
I will do it in 7m

**Comment by davidyaha** [comment]
If you'll force push to the same origin branch it will be the same PR

**Comment by Aetherall** [comment]
you mean force push to the mongo repo ?

**Comment by pradel** [comment]
I think here

**Comment by davidyaha** [comment]
No, force push to the branch here that this PR is made of `monorepo/mongo`

**Comment by Aetherall** [comment]
I am lost ><" 

I pull from the mongo-repository then I force push to this branch ?

How the commits will know that they apply to the package/database-mongo subfolder ?

I should lerna import again ?

Sorry for this

**Comment by pradel** [comment]
Maybe will be easier to delete this branch and create a new pr with lerna import :)

**Comment by Aetherall** [comment]
Closing in favor of #176

**Pull request closed without merging**